tests/manual-test-cases/Group18-VIC-UI/18-3-VIC-UI-NGC-tests.robot

```diff
@@ -56,6 +63,16 @@ Run Ngc Tests Project
 
 
 *** Keywords ***
+Update Config To Use Webserver For Plugin Testing
```

Comment:
I don't have a good view into the larger picture here, but this looks like it is something that needs to be pulled out into a UI-Util.robot and you have a lot of other repeated code that should probably be pulled into a unified library to make these tests more readable as well.